entente/test_rigid_transform.py

```diff
@@ -87,6 +88,9 @@ def test_rigid_rotation_single_point():
     np.testing.assert_array_almost_equal(a.dot(R), b)
 
 
+@pytest.mark.skipif(
+    sys.implementation.name == "cpython", reason="failing in CI when numpy<1.19.3"
```

Comment:
Shouldn't this condition match the reason? Something like `env_flag('CI')`. And isn't it newer numpy's that are the problem?